tests/testthat/test-decision_automation.R

```diff
@@ -0,0 +1,95 @@
+test_that("decision_automation works", {
+  # delete app DB if exists to ensure clean test
+  app_db_loc <- test_path("test-apps", "decision_automation-app", "database.sqlite")
+  if (file.exists(app_db_loc)) {
+    file.remove(app_db_loc)
+  }
+  
+  # copy in already instantiated database to avoid need to rebuild
+  # this is a database that has been built via inst/testdata/upload_format.csv
+  test_db_loc <- app_sys("testdata", "decision_automation_ex1.sqlite")
+  file.copy(
+    test_db_loc,
+    app_db_loc
+  )
+  
+  app <- shinytest2::AppDriver$new(test_path("test-apps", "decision_automation-app"))
+  
+  # Check that reactive values are loaded correctly
+  # Check datatable table
+  expected <- structure(list(decision = c("Insignificant Risk", "Severe Risk"), 
```

Comment:
no need to change the implementation, but this would be a place to consider using a [snapshot test](https://testthat.r-lib.org/articles/snapshotting.html) to avoid the need to `dput` the expected object in the test code itself.

i believe you can use the in-built `app$expect_values()` function to accomplish this:

https://rstudio.github.io/shinytest2/reference/AppDriver.html#testthat-wrapp

Reply:
I have had bad luck using `app$expect_values()` for this project. I can't seem to get it to work, hence utilizing `dput()` so much.